app/assets/javascripts/modules/track-selected-taxons.js

```diff
@@ -0,0 +1,28 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function TrackSelectedTaxons (module) {
+    this.form = module
+    this.requestPath = this.form.getAttribute('data-request-path')
+  }
+
+  TrackSelectedTaxons.prototype.init = function () {
+    this.form.addEventListener('submit', this.onSubmitListener.bind(this))
+  }
+
+  TrackSelectedTaxons.prototype.onSubmitListener = function (e) {
+    var requestPath = this.requestPath
+    var selectedItems = this.form.querySelectorAll('.miller-columns-selected__list .miller-columns-selected__list-item')
+
+    Array.from(selectedItems, function(item) {
+      var taxonPath = Array.from(item.querySelectorAll('.govuk-breadcrumbs__list-item'), function(crumb) {
```

Comment:
`Array.from` [isn't supported by IE11](https://caniuse.com/mdn-javascript_builtins_array_from) – is this going to be a problem? I'm not sure if this method gets polyfilled, or if we're yet at the stage where we can drop IE11 support?

Reply:
Updated